src/screens/Landing/HeroSection/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./style.scss";
+import rightbranch from "./assets/right.png";
+import leftbranch from "./assets/left.png";
+
+export default () => (
+  <div className='hero-section'>
+    <div className='part-1'>
```

Comment:
Fixed some merge conflicts on this. 

Additionally, It would be better to have a more semantic name here than part-1. Maybe hero-container or hero-wrapper. Additionally to prevent naming collisions. all of the tags should probably have class names. Avoid using <br> if possible, you could instead set a max-width on the paragraph. 

For part 2 it could be named info section or an aside.